.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [ ruby-head, '3.0', '2.7', '2.6' ]
+        ruby: [ ruby-head, '3.1', '3.0', '2.7' ]
```

Comment:
Why the `2.6` drop? I added it to support latest jruby.

Reply:
Oh I can add it back, I thought it was unsupported.